feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -4,7 +4,8 @@ $sapFioriContent_LabelColor: #6a6d70;
 
 %cx-customer-list {
   .cx-modal-content {
-    min-height: 424px;
+    max-height: 100%;
+    overflow-y: auto;
```

Comment:
Not to be a nit, but should we remove this `overflow-y` too? Does it cause double scroll bars?